tls/s2n_server_key_exchange.c

```diff
@@ -197,7 +192,9 @@ int s2n_kem_server_key_recv_parse_data(struct s2n_connection *conn, struct s2n_k
     const struct s2n_cipher_suite *cipher_suite = conn->secure->cipher_suite;
     const struct s2n_kem *match = NULL;
     S2N_ERROR_IF(s2n_choose_kem_with_peer_pref_list(cipher_suite->iana_value, &kem_data->kem_name, kem_preferences->kems,
-                                                    kem_preferences->kem_count, &match) != 0, S2N_ERR_KEM_UNSUPPORTED_PARAMS);
+                         kem_preferences->kem_count, &match)
+                    != 0,
+            S2N_ERR_KEM_UNSUPPORTED_PARAMS);
```

Comment:
I still hate this style of line wrapping. Why does the "!= 0" have to be on its own line?

If this is the price of formatting I can live with it, but it's really unpleasant.